Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/ArUcoMarkerAnchor.cs

```diff
@@ -15,15 +15,19 @@
 
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.HolographicCamera
 {
-    [RequireComponent(typeof(WorldAnchorManager))]
-    [RequireComponent(typeof(TCPConnectionManager))]
-    public class HolographicCameraOriginAnchor : MonoBehaviour
+    /// <summary>
+    /// Represents an object that can detect an ArUco marker and persist and restore a WorldAnchor
+    /// to persist the location of that marker across sessions.
+    /// </summary>
+    public class ArUcoMarkerAnchor : MonoBehaviour, ICommandHandler
```

Comment:
>ArUcoMarkerAnchor [](start = 17, length = 17)

Nothing about this is ArUco specific, we should change this to MarkerAnchor so that it can be used with qr codes/HoloLens 2 as well.